tests/base.py

```diff
@@ -0,0 +1,256 @@
+import unittest
```

Comment:
@dbshah1212 This code is already added as part of (https://github.com/singer-io/tap-pipedrive/pull/90/files)
Why is it added again here?